beacon-chain/powchain/service.go

```diff
@@ -432,19 +432,10 @@ func (s *Service) dialETH1Nodes(endpoint network.Endpoint) (*ethclient.Client, *
 		closeClients()
 		return nil, nil, err
 	}
-	nID, err := httpClient.NetworkID(s.ctx)
```

Comment:
Don't delete this if you want to merge to kintsugi. The intend for that branch is to mimic production as much as possible. It will make the diff between `kintsugi` and `develop` confusing